#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_unset-python-path.dpatch by  <andrewross@users.sourceforge.net>
##
## DP: No need to explicitly set python path with debian since the
## DP: default path is correct. This ensure the examples work with
## DP: multiple version of python.

@DPATCH@

Index: plplot-5.9.9/examples/python/CMakeLists.txt
===================================================================
--- plplot-5.9.9.orig/examples/python/CMakeLists.txt	2012-02-28 11:40:09.000000000 +0000
+++ plplot-5.9.9/examples/python/CMakeLists.txt	2012-02-28 11:42:31.000000000 +0000
@@ -241,7 +241,7 @@
     @ONLY
     )
 
-  set(python_location1 ${PYTHON_INSTDIR})
+  set(python_location1)
   set(python_location2)
   set(python_location3)
   set(python_location4)
